packages/astroport_periphery/src/lockdrop_pcl.rs

```diff
@@ -296,15 +286,12 @@ pub struct State {
 pub struct PoolInfo {
     pub lp_token: Addr,
     pub amount_in_lockups: Uint128,
-    // pub migration_info: Option<MigrationInfo>,
     /// Share of total NTRN incentives allocated to this pool
     pub incentives_share: Uint128,
     /// Weighted LP Token balance used to calculate NTRN rewards a particular user can claim
     pub weighted_amount: Uint256,
-    /// Ratio of Generator NTRN rewards accured to astroport pool share
-    pub generator_ntrn_per_share: Decimal,
-    /// Ratio of Generator Proxy rewards accured to astroport pool share
-    pub generator_proxy_per_share: RestrictedVector<AssetInfo, Decimal>,
+    /// Ratio of Generator rewards accured to astroport pool share
+    pub generator_rewards_per_share: RestrictedVector<AssetInfo, Decimal>,
     /// Boolean value indicating if the LP Tokens are staked with the Generator contract or not
     pub is_staked: bool,
```

Comment:
i dont think we really need this field, we never set it to false. It would make sence in case, the lockdrop could withdraw tokens from the incentives contract and keep them, but there is no such handler.

Reply:
Let's leave it as it is for now, there are a lot of places where it's used

Reply:
Done